html/index.html

```diff
@@ -0,0 +1,142 @@
+            일상의 모든 물건을 <br />
+            거래해보세요
+          </h2>
+          <a href="/html/items.html" class="contentBtn">구경하러 가기</a>
+        </div>
+        <img
+          src="/images/section_panda.png"
+          alt="배너 사진"
+          class="contentImg"
+        />
+      </section>
+
+      <!-- main contents -->
+      <section class="mainContents">
+        <!-- content1 -->
+        <div class="mainContent">
+          <img
+            src="/images/main_content_img1.png"
+            alt="메인사진1"
+            class="mainContentImg"
+          />
+          <div class="imageText">
+            <h4>Hot item</h4>
+            <h2>
+              인기 상품을 <br />
+              확인해 보세요
+            </h2>
+            <p>가장 HOT한 중고거래 물품을 판다 마켓에서 확인해보세요</p>
+          </div>
+        </div>
+
+        <!-- content2 -->
+        <div class="mainContent_even">
+          <div class="imageText">
+            <h4>Search</h4>
+            <h2>
+              구매를 원하는 <br />
+              상품을 검색하세요
+            </h2>
+            <p>구매하고 싶은 물품은 검색해서 쉽게 찾아보세요</p>
+          </div>
+          <img
+            src="/images/main_content_img2.png"
+            alt="메인사진2"
+            class="mainContentImg"
+          />
+        </div>
+
+        <!-- content3 -->
+        <div class="mainContent">
+          <img
+            src="/images/main_content_img3.png"
+            alt="메인사진3"
+            class="mainContentImg"
+          />
+          <div class="imageText">
+            <h4>Register</h4>
+            <h2>
+              판매를 원하는 <br />
+              상품을 등록하세요
+            </h2>
+            <p>어떤 물건이든 판매하고 싶은 상품을 쉽게 등록하세요</p>
+          </div>
+        </div>
+      </section>
+
+      <!-- bottom banner -->
+      <section class="contents">
+        <div class="content">
+          <h2>
+            믿을 수 있는 <br />
+            판다마켓 중고거래
+          </h2>
+        </div>
+        <img
+          src="/images/footer_content_panda.png"
+          alt="하단배너 사진"
+          class="contentImg"
+        />
+      </section>
+    </main>
+
+    <!-- footer -->
+    <footer id="footer">
+      <div class="footerContents">
+        <span>©codeit - 2024</span>
+        <div class="footerPolicy">
+          <a href="/html/privacy.html">Privacy Policy</a>
+          <a href="/html/faq.html">FAQ</a>
+        </div>
+
+        <div class="footerSns">
+          <a href="https://www.facebook.com/" target="_blank">
+            <img src="/images/svg/facebook-logo.svg" alt="페이스북" />
+          </a>
+          <a href="https://x.com/?lang=ko" target="_blank">
+            <img src="/images/svg/twitter-logo.svg" alt="x" />
+          </a>
+          <a href="https://www.youtube.com/" target="_blank">
+            <img src="/images/svg/youtube-logo.svg" alt="유튜브" />
+          </a>
+          <a href="https://www.instagram.com/" target="_blank">
+            <img src="/images/svg/instagram-logo.svg" alt="인스타그램" />
+          </a>
+        </div>
```

Comment:
💊 제안
이 영역은 SNS들의 모음이라고 할 수 잇습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footerSns">
          <li>
            <a href="https://www.facebook.com/" target="_blank">
              <img src="/images/svg/facebook-logo.svg" alt="페이스북" />
            </a>
          </li>
        </ul>
```